### PR TITLE
Approve transactions

### PR DESCRIPTION
**Pull request by iVuong**

### CODE REVIEW

**Comment by thatniggafacts** [comment]
Let gets this to work

File: banking_system/transaction_management/templates/transaction_management/pendingCashierChecks.html
Comment (zlmonroe) [inline banking_system/transaction_management/templates/transaction_management/pendingCashierChecks.html]: remove this :-P

File: banking_system/transaction_management/templates/transaction_management/pendingFundTransfers.html
Comment (zlmonroe) [inline banking_system/transaction_management/templates/transaction_management/pendingFundTransfers.html]: Remove this too :-P

**Review by zlmonroe — DISMISSED** [review]
Just two log lines. I'm going to wait on approval until you look over mine though

**Review by zlmonroe — APPROVED** [review]
This is good. There is an earlier problem from credit where it asked for an email we should fix.

**Pull request merged**
